package/MDAnalysis/coordinates/core.py

```diff
@@ -51,7 +51,8 @@ def get_reader_for(filename, permissive=False, format=None):
         filename of the input trajectory or coordinate file
     permissive : bool
```

Comment:
Will do, thanks.
On Wed, Apr 13, 2016 at 4:17 PM Oliver Beckstein <notifications@github.com>
wrote:

> In package/MDAnalysis/coordinates/core.py
> <https://github.com/MDAnalysis/mdanalysis/pull/812#discussion_r59642201>:
>
> > @@ -51,7 +51,8 @@ def get_reader_for(filename, permissive=False, format=None):
> >          filename of the input trajectory or coordinate file
> >      permissive : bool
>
> remove the whole docs for permissive, users should not need to know about
> anymore.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/MDAnalysis/mdanalysis/pull/812/files/02b66897f7f8c17ac0e0c6cd28a9f5a50a3a31d2#r59642201>
>
-- 
Have a wonderful day,
John J. Detlefs